README.md

```diff
@@ -1,2 +1,48 @@
 # Hello_Microverse-
 Hello microverse project 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+# Hello Microverse
+
+> Hello-Microverse.
+This is a project to display "Hello Microverse".
+
+
+## Built With
+- HTML
+- CSS
+## Live Demo (if available)
+
+## Getting Started
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Please make sure to customize the readme file. Currently, I can see some headings that are not used:

```
 ### Run tests
### Deployment
``` 

And paragraphs that are not relevant:

```
**This is an example of how you may give instructions on setting up your project locally.**
**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
To get a local copy up and running follow these simple example steps.

```